### PR TITLE
noFeedTour working and search 'page' by using cookies

### PR DESCRIPTION
**Pull request by Feikoritsema**
Working

### CODE REVIEW

**Comment by Feikoritsema** [comment]
Could this be merged ? @mircealungu

File: src/setup.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 setuptools.setup(
     name="umr",
-    version="0.8.3",
+    version="0.9.0",
```
Comment (mircealungu) [inline src/setup.py]: oki, i think we need to update the `change_version_number.sh` to keep the `setup.py` in sync with the `package.json`. There we have version 0.9.2 and here 0.9.0. I don't think it makes sense!

Comment (mircealungu) [reply src/setup.py]: oki, fixed in: https://github.com/zeeguu-ecosystem/Unified-Multilanguage-Reader/commit/2e191a86e0a64b54239ec19c8346f6852f80a7c5

File: src/umr/static/scripts/app/subscription/main.js [hunk 2]
```diff
@@ -125,9 +133,38 @@ function showSearchNotification(input){
...
+    let d = new Date();
+    // We make it expire after one day
+    d.setTime(d.getTime() + (24*60*60*1000));
+    let expires = "expires="+ d.toUTCString();
+    document.cookie = SEARCH_COOKIE_NAME + "=" + input + ";" + expires + ";path=/";
+}
+
+function getSearchCookie(){
+    let decodedCookie = decodeURIComponent(document.cookie);
+    let ca = decodedCookie.split(';');
```
Comment (mircealungu) [inline src/umr/static/scripts/app/subscription/main.js]: sorry, what does `ca` stand for? can you use a more self-explaining name?

File: src/umr/static/scripts/app/subscription/main.js [hunk 3]
```diff
@@ -125,9 +133,38 @@ function showSearchNotification(input){
...
+    let nameEq = SEARCH_COOKIE_NAME + "=";
+    for(let i = 0; i <ca.length; i++) {
+        let c = ca[i];
+        while (c.charAt(0) == ' ') {
+            c = c.substring(1);
+        }
```
Comment (mircealungu) [inline src/umr/static/scripts/app/subscription/main.js]: this is a trim left here, right? 
can't you use the jquery version instead? 
https://stackoverflow.com/a/498979/1200070

**Pull request closed without merging**